modules/appeals_api/app/services/appeals_api/remove_pii.rb

```diff
@@ -41,13 +41,13 @@ def valid_form_type?
     end
 
     def records_to_be_expunged
-      @records_to_be_expunged ||=
-        form_type.where.not(form_data_ciphertext: nil)
-                 .or(
-                   form_type.where.not(
-                     auth_headers_ciphertext: nil
-                   )
-                 ).pii_expunge_policy
+      @records_to_be_expunged ||= if Flipper.enabled?(:decision_review_updated_pii_rules)
```

Comment:
I put this behind a flag partially so that it could be easily toggled off if something is wrong, but the primary reason was that otherwise there would be so many lines to change (in tests and models) that it seemed like I wouldn't be able to fit these changes into a single PR.